EIPS/eip-5170.md

```diff
@@ -0,0 +1,144 @@
+---
+eip: 5170
+title: Asserting authenticity of Client Script for Token Contracts
+description: Provide a method to assert and revoke the authenticity of an ERC5169 client script for token contracts
+author:  Weiwu (@weiwu-zhang), Tore Frederiksen (@jot2re)
+discussions-to: 
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-05-03
+requires: 5169
```

Comment:
Since this is very much linked to that EIP, I would recommend you merge this EIP with that one. Technically, this EIP cannot be merged until all EIPs in the `requires` list reach final.

Reply:
My general feeling is that if an EIP *can* be split into multiple EIPs it *should* be split into multiple EIPs.  If this is an extension that could be split out, I think that is the correct way forward.

> Technically, this EIP cannot be merged until all EIPs in the requires list reach final.

The rule is that an EIP can't pass the status of any EIP it depends on.  So if 5169 is in review, this can be in Draft or Review but not Last Call of Final.  This allows someone to walk multiple EIPs through the process at the same time when they have a dependency association between them.